### PR TITLE
Completed all the tasks for Homework 1 and 2 (Reading user input & BMI calculated and second person)

### PR DESCRIPTION
**Pull request by Sakariya-Mohumed**
The program can read a user's input which includes the following:

- The user's first name.
- The user's last name.
- The user's age, weight and height 

This will then be printed on the console as:
"John Smith is 29 years old, their weight is 90 and their height is 189"

The second task consisted of calculating the BMI index using the user's input which has also been done.
The BMI will be then printed as follows:
"John, your BMI value is [BMI Value]"

### CODE REVIEW

**Comment by Sakariya-Mohumed** [comment]
I have realised that I have not repeated it for one other person. Will get that done by the end of today.

**Comment by Sakariya-Mohumed** [comment]
Task 3 has been done where the process needs to happen twice for another user

File: Src/BootCamp.Chapter/Program.cs [hunk 1]
```diff
@@ -2,11 +2,81 @@
...
+            // The string need to be concatanated for a final message
+            Console.WriteLine(Firstname + " " + Lastname + " is" + " " + age + " years old, " + "their weight is " + weight + " and their height is " + height);
+
+            //-----Task 2-----//
+
+            /*BMI needs to be calculated from what the user has inputted
+             * The formula of BMI= weight divided by height squared
+             * The weight and height need to be converted to double */
+
+            var weightBmi = Convert.ToDouble(weight);
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs:40]: It's simply weight. I'd prefer to see a disinction like weightInput, weight.

File: Src/BootCamp.Chapter/Program.cs [hunk 2]
```diff
@@ -2,11 +2,81 @@
...
+            Console.WriteLine("Please enter your height: ");
+            string User2height = Console.ReadLine();
+
+            Console.WriteLine(User2name + " " + User2surname + " is" + " " + User2age + " years old, " + "their weight is " + User2weight + " and their height is " + User2height);
+
+            var weightbmi_2 = Convert.ToDouble(User2weight);
+            var heightBmi_2 = Convert.ToDouble(User2height) / 100;
+
+            var heightsquared_2 = heightBmi_2 * heightBmi_2;
+            var Bmi_2 = weightbmi_2 / heightsquared_2;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs:77]: No underscores in naming.
Using camelCase.

**Review by Almantask — APPROVED** [review]
Looks good

**Pull request closed without merging**